Animal-sound/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+
+// interfata IAnimal
+interface IAnimal
+{
+    string Nume { get; }
```

Comment:
engleza este limba de baza in programare; nu cred ca ai vrea sa citesti cod scris in germana, rusa sau indiana.